Integrations/VMRay/VMRay.py

```diff
@@ -490,6 +490,56 @@ def get_submission(submission_id):
     response = http_request('GET', url_suffix=suffix)
     return response
 
+def check_file_command():
+    file_hash_input = demisto.args().get('file')
+    file_hashes = file_hash_input.split(',')
+    for file_hash in file_hashes:
+        if len(file_hash) != 32 and len(file_hash) != 40 and len(file_hash) != 64:
+            return_error('Invalid input hash')
+        hash_type = 'md5'
+        if len(file_hash) == 40:
+            hash_type = 'sha1'
+        if len(file_hash) == 64:
+            hash_type = 'sha256'
+
+        raw_response = check_file(file_hash, hash_type)
+        data = raw_response.get('data')
+
+        if len(data) == 0:
+            return_outputs(
+                'No submission found in VMRay for hash : {}'.format(file_hash),
+                {},
+            )
+
+        for item in data:
+
+            entry = dict()
+            entry['SampleID'] = item.get('sample_id')
+            entry['FileName'] = item.get('sample_filename')
+            entry['MD5'] = item.get('sample_md5hash')
+            entry['SHA1'] = item.get('sample_sha1hash')
+            entry['SHA256'] = item.get('sample_sha256hash')
+            entry['SSDeep'] = item.get('sample_ssdeephash')
+            entry['Severity'] = SEVERITY_DICT.get(item.get('sample_severity'))
+            entry['Type'] = item.get('sample_type')
+            entry['Created'] = item.get('sample_created')
+            entry['Classification'] = item.get('sample_classifications')
+            scores = dbot_score_by_hash(entry)
+
+            entry_context = {
```

Comment:
need also to return to `File` path according to [context standard](https://github.com/demisto/content/tree/master/docs/context_standards#file)

Reply:
This part of the function was taken straight from the get_sample_command() function.